modules/base-commerce/composables/useLocation.ts

```diff
@@ -0,0 +1,15 @@
+import { onMounted, ref } from 'vue'
+
+export default () => {
+  const location = ref(null)
+
+  onMounted(() => {
+    if (!('geolocation' in navigator)) return
+
+    navigator.geolocation.getCurrentPosition((position: any) => (location.value = position.coords))
+  })
+
+  return {
```

Comment:
Composable is named useLocation so i would expect the composable to return location object. Not object containing location and possibly other stuff. Also location can contain many things, coordinates being on of the so I would change the return like this:

```
return reactive({
  coordinates,
})
```

Coordinates being the coords field